lib/Driver/Driver.cpp

```diff
@@ -1350,6 +1350,18 @@ void Driver::buildOutputInfo(const ToolChain &TC, const DerivedArgList &Args,
              "unknown -g<kind> option");
   }
 
+  if (const Arg *A = Args.getLastArg(options::OPT_debug_info_format)) {
+    if (A->containsValue("dwarf"))
+      OI.DebugInfoKind = IRGenDebugInfoKind::DwarfTypes; // Or `::Normal`?
+    else if (A->containsValue("codeview"))
+      OI.DebugInfoKind = IRGenDebugInfoKind::CodeViewTypes;
+    else {
+      Diags.diagnose(SourceLoc(), diag::error_invalid_arg_value,
+                     A->getAsString(Args), A->getValue());
+      return;
```

Comment:
I think that this would be nicer as a `StringSwitch`.

Reply:
I don't think I can use `StringSwitch` after moving `CodeViewTypes` out of `IRGenDwarfDebugInfoKind`.